packages/toolkit/src/extensionWeb.ts

```diff
@@ -34,6 +37,14 @@ export async function activate(context: vscode.ExtensionContext) {
     }
 }
 
+/**
+ * The browserfied version of os does not have a `version()` method,
+ * so we patch it.
+ */
+function patchOsVersion() {
+    ;(os.version as any) = () => '1.0.0'
```

Comment:
I assume this is for library code. Is there a particular dependency that fails if this doesn't exist? Does the browserified "os" return something actually meaningful for the other fields?

Reply:
> `shouldTriggerFromClassifier()` using `os.version()`

That's code that we own, so perhaps we should abstract `os` and use the abstraction from there, instead of shimming. That will avoid surprising situations where `os.version` is used to make a wrong decision.